code/game/machinery/fabricator.dm

```diff
@@ -0,0 +1,569 @@
+#define FABRICATOR_MAIN_MENU       1
+#define FABRICATOR_CATEGORY_MENU   2
+#define FABRICATOR_SEARCH_MENU     3
+#define FABRICATOR 5
+/obj/machinery/fabricator
```

Comment:
You copypasted the entirety of the autolathe code but renamed it. why didn't you just make this a child of the autolathe?